src/Analysis/Engine/Impl/Intellisense/AnalysisQueue.cs

```diff
@@ -34,13 +34,16 @@ internal sealed class AnalysisQueue : IDisposable {
         private readonly HashSet<IGroupableAnalysisProject> _enqueuedGroups;
         private readonly PriorityProducerConsumer<QueueItem> _ppc;
         private readonly Task _consumerTask;
+        private readonly ManualResetEventSlim _queueEnabled = new ManualResetEventSlim(true);
 
         public event EventHandler AnalysisStarted;
         public event EventHandler AnalysisComplete;
         public event EventHandler AnalysisAborted;
         public event EventHandler<UnhandledExceptionEventArgs> UnhandledException;
 
         public int Count => _ppc.Count;
+        public void Start() => _queueEnabled.Set();
```

Comment:
It looks like we can replace pair of `Start` and `Stop` with `IDisposable Pause` using `CountdownDisposable` from RTVS: https://github.com/Microsoft/RTVS/blob/master/src/Common/Core/Impl/Disposables/CountdownDisposable.cs

Reply:
OK. I did it in tests, will move to main line.